spec/support/step_nav_steps.rb

```diff
@@ -79,8 +79,8 @@ def given_there_is_a_published_step_by_step_page_with_unpublished_changes
     expect(@step_by_step_page.status[:name]).to eq 'unpublished_changes'
   end
 
-  def given_there_is_a_step_by_step_page_assigned_to_me
-    @step_by_step_page = create(:step_by_step_page_with_navigation_rules, assigned_to: stub_user.name)
+  def given_I_am_assigned_to_a_live_step_by_step_page_with_unpublished_changes
+    @step_by_step_page = create(:published_step_by_step_page, draft_updated_at: Time.zone.now, assigned_to: stub_user.name)
```

Comment:
Why was this method changed? How does it related to setting the change note to "First published..."?

Reply:
The "User publishes and then makes more changes to a step by step page" scenario started to fail, because it was trying to assert:

`then_there_should_be_a_change_note "Minor update published by #{stub_user.name}"`

But the `create(:step_by_step_page_with_navigation_rules, assigned_to: stub_user.name)` creates a step by step without setting `published_at`, meaning it was being treated as a first publish, meaning the change note became "First published by Test author", therefore failing the change note assertion.

So I've now made sure it calls a factory method that creates a _published_ step by step, but with a draft (so that there is something to publish), and have renamed the method to make it clearer what it's doing.

Reply:
Thanks for explaining! Can you add a version of that to the commit note?